apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -563,21 +557,23 @@ private void processAsync(
   }
 
   /**
-   * Checks if the record is a beneficiary and has a enrollment reference year that is either <code>
-   * null</code> or is from 2022. This is to handle special filtering while CCW fixes an issue and
-   * should be temporary.
+   * Checks if the record should be filtered by checking if it is a beneficiary, the flag to filter
+   * items is on, the bene has a non-{@code null} enrollment reference year and is not equal to
+   * 2022. This is to handle special filtering while CCW fixes an issue and should be temporary.
    *
    * @param rifRecordEvent the {@link RifRecordEvent} to check
-   * @return {@code true} if the record is a beneficiary and has an enrollment year that is either
-   *     <code>null</code> or 2022
+   * @return {@code true} if the record is a beneficiary and has an enrollment year that is non-
+   *     <code>null</code> and not 2022, and the flag to filter such beneficiaries is set to {@code
+   *     true}
    */
-  private boolean isBeneficiaryWithNullOr2022Year(RifRecordEvent<?> rifRecordEvent) {
-    if (rifRecordEvent.getRecord() instanceof Beneficiary) {
+  private boolean shouldBeFiltered(RifRecordEvent<?> rifRecordEvent) {
```

Comment:
Open to name adjustments like `shouldBeSkipped` or something, but wanted the name to be a positive declaration of whats happening since before this method was being negated always which was confusing.

Reply:
certainly don't have any objection to name.